src/components/Garden/ProposalDetail/ProposalDetail.js

```diff
@@ -622,7 +622,16 @@ const Amount = ({
                 {requestToken.symbol}
               </span>
               <Help hint="">
-                Converted to {requestToken.symbol} at time of execution
+                Converted to {requestToken.symbol} at time of execution. For
```

Comment:
Probably we should add the same message to the `ProposalInfo`. What do you think?

Reply:
Ah yeas good catch